README.md

```diff
@@ -299,20 +299,23 @@ or an empty string. Conditional votes are a comment representing a vote.
 In order to perform an action based on resolution of one of more conditional votes,
 a player must post a comment describing the resolution before performing said action.
 
-**345** *Dependency Hell*
+**368** *Dependency Hell*
 
 The owner of a PR may declare one dependency of this PR (henceforth "child") on another
 PR (henceforth "parent"), by including the text
 "This PR is dependent on #<number of parent>" in an unedited post. Any votes above
 this post are voided. A child PR must be of a branch that branches off the
-branch of the parent. Both PRs must be on master.
+branch of the parent at the moment of declaring the dependency. Both PRs must be on master.
 
 A vote in favour on a child implies a vote in favour on its parent. This vote shall be read
 as a comment representing a vote in favour on the parent just before merging 
 for the purposes of other rules.
 
 A child may be formulated as if the parent is the current ruleset.
 
+If a commit that invalidates votes is added on a parent, any vote that
+implied one on that parent at that time is invalidated.
```

Comment:
This line specifies that because A gets changed, the vote on B gets invalidated. (So it solves the problem you're describing by undoing the vote on B.)